src/main/java/com/almostreliable/ponderjs/PonderRegistryEventJS.java

```diff
@@ -16,4 +19,17 @@ public PonderBuilderJS create(String name, IngredientJS ingredient) {
         }
         return new PonderBuilderJS(name, ingredient.getVanillaItems());
     }
+
+    public void printParticleNames() {
+        StringBuilder sb = new StringBuilder();
+        sb.append("\n").append("### Particles ###").append("\n");
+        //noinspection ConstantConditions
+        ForgeRegistries.PARTICLE_TYPES.getValues()
+                .stream()
+                .filter(p -> p instanceof SimpleParticleType)
```

Comment:
replace this filter with a method reference `SimpleParticleType.class::isInstance`